src/dualscreen.tsx

```diff
@@ -42,14 +46,18 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
     inverse: false,
     layout: Layout.PIP,
     childSizePercentage: 30,
+    childAspectRatio: {
+      width: 16,
+      height: 9,
+    },
     position: Position.BottomRight
   };
 
   constructor(name: string, player: any, config: DualScreenConfig) {
     super(name, player, config);
     this._player = player;
     this.secondaryKalturaPlayer = this._createSecondaryPlayer();
-    this._imagePlayer = new ImagePlayer(this._setMode);
+    this._imagePlayer = new ImagePlayer(this._onActiveSlideChanged);
```

Comment:
this change should be applied for `SideBySideWrapper` too

Reply:
please see https://github.com/kaltura/playkit-js-dual-screen/blob/3c56efe4c72c66615bc9e246d7356d6b95afb7d7/src/dualscreen.tsx#L345